plugins/extend-give-wp/src/support/load-assets.php

```diff
@@ -0,0 +1,90 @@
+<?php
+/**
+ * Load the assets
+ *
+ * @package     spiralWebDB\ExtendGiveWP
+ * @since       1.0.0
+ * @author      Robert A. Gadon
+ * @link        https://spiralwebdb.com
+ * @license     GNU General Public License 2.0+
+ */
+
+namespace spiralWebDB\ExtendGiveWP;
+
+add_action( 'give_pre_form', __NAMESPACE__ . '\render_form_featured_image_and_caption' );
+/*
+ * Render donation form featured image and caption.
+ *
+ * @since 1.0.0
+ * @param int $_id  Post ID of the donation form.
+ *
+ * @return void
+ */
+function render_form_featured_image_and_caption( $_id = 1349 ) {
+	// Get the post object for the attachment via get_post().
+	$post    = get_post( 1411 );
+	$postID  = $post->ID;
+
+	require _get_plugin_dir() . '/src/views/donation-form-1349/featured-image-view.php';
```

Comment:
Why a specific post ID in the view name?

Reply:
@hellofromtonya 
> Why a specific post ID in the view name?

Because the plugin customizations were targeting the Give donation form with `$_id` = 1349.

On a separate but related matter, the custom callback could pass `$attachment_id` as an argument, and the value of that parameter could be passed from a custom configuration. 

The view called within the callback passes `$attachment_id` as an argument to the WordPress API function `wp_get_attachment_image()` ( located at `/wp-includes/media.php` on L#983 ). That would allow a refactor within the callback where we could get rid of lines 25 & 26 above.